test/util/data_test.py

```diff
@@ -45,7 +46,10 @@ class FloatSubclass(float):
     self.assertIs(type(converted['unicode']), str)
     self.assertIs(type(converted['int']), int)
     self.assertIs(type(converted['float']), float)
-    self.assertIs(type(converted['long']), int)
+    if sys.version_info[0] < 3:
```

Comment:
There are a couple of idioms that are preferable to testing the Python version explicitly. See http://python-future.org/compatible_idioms.html#long-integers

pseudocode:
```
from builtins import int
isInstance(converted['long'], int)
```